src/intTest/resources/fhir/screening.fhir.json

```
@@ -0,0 +1,162 @@
+          }
+        ],
+        "gender": "male",
+        "birthDate": "1938-02-06",
+        "address": [
+          {
+            "postalCode": "TQ14 4QJ"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:bcbb255a-fd29-4811-97c4-b5d77e8ed731",
+      "resource": {
+        "resourceType": "Practitioner",
+        "id": "bcbb255a-fd29-4811-97c4-b5d77e8ed731",
+        "name": [
+          {
+            "text": "BCSP Hub Manager"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:fc050c50-8fbb-4b91-a9eb-509d4c6f745d",
+      "resource": {
+        "resourceType": "Organization",
+        "id": "fc050c50-8fbb-4b91-a9eb-509d4c6f745d",
+        "type": [
+          {
+            "coding": [
+              {
+                "code": "dept",
+                "display": "UNKNOWN"
+              }
+            ]
+          }
+        ],
+        "name": "Southern Bowel Cancer Screening Programme Hub"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:cb5babe7-1ce3-4ce5-9206-08f5145b745b",
+      "resource": {
+        "resourceType": "DiagnosticReport",
+        "id": "cb5babe7-1ce3-4ce5-9206-08f5145b745b",
+        "identifier": [
+          {
+            "value": "34/1"
+          }
+        ],
+        "status": "unknown",
+        "code": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "721981007",
+              "display": "Diagnostic studies report"
+            }
+          ]
+        },
+        "subject": {
+          "reference": "urn:uuid:68f3ed2a-9f5f-4d2d-84a9-0c4aaa5529de"
+        },
+        "issued": "2009-11-10T11:45:00.000+00:00",
+        "performer": [
+          {
+            "actor": {
+              "reference": "urn:uuid:fc050c50-8fbb-4b91-a9eb-509d4c6f745d"
+            }
+          }
+        ],
+        "specimen": [
+          {
+            "reference": "urn:uuid:ad79dabe-2842-4a74-8f09-72cd398fcdd4"
+          }
+        ],
+        "result": [
+          {
+            "reference": "urn:uuid:77251914-ba65-4fc7-8467-d00c1c4b97d1"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:ad79dabe-2842-4a74-8f09-72cd398fcdd4",
+      "resource": {
+        "resourceType": "Specimen",
+        "id": "ad79dabe-2842-4a74-8f09-72cd398fcdd4",
+        "type": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "display": "Faeces"
+            }
+          ]
+        },
+        "subject": {
+          "reference": "urn:uuid:68f3ed2a-9f5f-4d2d-84a9-0c4aaa5529de"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:77251914-ba65-4fc7-8467-d00c1c4b97d1",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "77251914-ba65-4fc7-8467-d00c1c4b97d1",
+        "status": "unknown",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
```

From looking at the Screening edifact document, I think this should be using SNOMED:

```suggestion
              "system": "http://snomed.info/sct",
```

---

This may need addressed in a separate story. We use the system value "http://loinc.org" across all our FHIR files, so I wouldn't want to change all them

---

👍  We'll need a separate story to address the use of `system` in other FHIR files, as we use "http://loinc.org" elsewhere which I believe isn't correct

---

Created here, https://gpitbjss.atlassian.net/browse/NIAD-1180